.github/workflows/nomad.yaml

```diff
@@ -97,6 +98,22 @@ jobs:
           NOMAD_VAR_scenario-url: ${{ steps.get-download-url.outputs.download-url }}
           NOMAD_VAR_run-id: ${{ github.run_id }}
         run: |-
-          nix run --impure --inputs-from . nixpkgs#nomad -- job run nomad/jobs/${{ matrix.job-name || matrix.scenario-name }}.nomad.hcl
+          set -Eeuo pipefail
```

Comment:
Is `-E` needed? That's not one we normally set

Reply:
Removed